frontend/src/core/editor/components/connectedEditor.js

```diff
@@ -250,21 +250,27 @@ export default function connectedEditor<Object>(
             this.updateTranslation('', true);
         }
 
-        sendTranslation = (ignoreWarnings?: boolean, translation?: string) => {
+        sendTranslation = (ignoreWarnings?: boolean, translation?: string, simpleTranslation?: Translation) => {
             const props = this.props;
 
             if (props.editor.isRunningRequest || !props.selectedEntity || !props.locale) {
                 return;
             }
 
             const content = translation || props.editor.translation;
+            const simpleContent = simpleTranslation || props.editor.translation;
```

Comment:
Not sure `simpleTranslation` is a good name - it's not simple when it comes from the RichEditor. Maybe `editorTranslation`? And editorContent instead of `simpleContent`?